### PR TITLE
Enhance `README.md`

### PR DESCRIPTION
**Pull request by nabenabe0928**
## Motivation

This PR aims to enhance the README.md.

### CODE REVIEW

**Comment by nabenabe0928** [comment]
@not522 Could you review this PR?

**Comment by not522** [comment]
Sorry for my late response. How about setting all `<details>` to `open` so that we can see at a glance what the examples are? This will also allow searching in a web browser.

**Comment by nabenabe0928** [comment]
@not522 Thank you for the comment, I addressed your suggestion!

**Review by y0z — COMMENTED** [review]
Could you resolve conflicts?

**Comment by nabenabe0928** [comment]
@not522 @y0z 
I resolved the conflict!
Please take another look at README:)
https://github.com/nabenabe0928/optuna-examples/tree/enhance/improve-readme

**Review by y0z — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -56,118 +83,108 @@ The examples below provide codeblocks similar to the example above for various d
...
-* [Optimization of Hyperparameters for Stable-Baslines Agent](./rl/sb3_simple.py)
+* [Tracking Optimization Process with aim](./aim/aim_integration.py)
+* [Tracking Optimization Process with MLflow](./mlflow/keras_mlflow.py)
+* [Tracking Optimization Process with Weights & Biases](./wandb/wandb_integration.py)
+* [Optimization with Hydra](./hydra/simple.py)
+</details>
 
-### External projects using Optuna
+<details open>
+<summary>Real-World Optuna Examples by External Projects</summary>
```
Comment (not522) [inline README.md:157]: These links don't provide examples, so how about keeping the original text?

```suggestion
<summary>External projects using Optuna</summary>
```

Comment (nabenabe0928) [reply README.md:157]:
```suggestion
<summary>External Projects Using Optuna</summary>
```

**Review by not522 — APPROVED** [review]
LGTM

**Pull request merged**